src/Microsoft.TestPlatform.PlatformAbstractions/netcore/System/PlatformEnvironment.cs

```diff
@@ -28,6 +28,7 @@ public PlatformArchitecture Architecture
                 // preview 6 or later, so use the numerical value for now.
                 // case System.Runtime.InteropServices.Architecture.S390x:
                 (Architecture)5 => PlatformArchitecture.S390x,
+                (Architecture)6 => PlatformArchitecture.PPC64le,
```

Comment:
Is this value correct? Shouldn't this be `8`?  I am looking at https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Architecture.cs, and there's `LoongArch64` and `Armv6` between `S390x` and `PPC64le`.

Reply:
yes you are right. Thanks for catching that. I matched it with enum in VSTest code so counted it 6. Will correct.